apps/src/common.rs

```diff
@@ -1488,9 +1524,43 @@ impl HttpConn for Http3Conn {
                             },
                         };
 
-                    debug!(
-                        "Prioritizing request on stream {} as {:?}",
-                        stream_id, priority
+                    match self.h3_conn.take_last_priority_update(stream_id) {
+                        Ok(v) => {
+                            priority = v;
+                        },
+
+                        Err(quiche::h3::Error::Done) => (),
+
+                        Err(e) => error!(
+                            "{} error taking PRIORITY_UPDATE {}",
+                            conn.trace_id(),
+                            e
+                        ),
+                    }
+
+                    if !priority.is_empty() {
+                        headers.push(quiche::h3::Header::new(
+                            b"priority",
+                            priority.as_slice(),
+                        ));
+                    }
+
+                    #[cfg(feature = "sfv")]
+                    let priority =
+                        match quiche::h3::Priority::try_from(priority.as_slice())
+                        {
+                            Ok(v) => v,
+                            Err(_) => quiche::h3::Priority::default(),
+                        };
+
+                    #[cfg(not(feature = "sfv"))]
+                    let priority = quiche::h3::Priority::default();
+
+                    info!(
```

Comment:
`debug!()` is better?

Reply:
This is on the server, where receiving the priority event trigger, and receiving request headers are logged at info level. Having the server state what priority it is applying at info level seems helpful to me.